docsource/README.md

```diff
@@ -0,0 +1,22 @@
+The documentation in this directory is based on [Sphinx](http://www.sphinx-doc.org/en/stable/).
+
+
+The makefile and conf.py was generated using,
+
+ $ sphinx-quicksart
+
+There is no need to regenerate the makefile.
+
+
+Simply generate html documentation locally in the build dir, use
+
+ $ make html
```

Comment:
Can you add which directory you should be in? The multiple Makefiles threw me off atleast.